build/references/command-line-interface.md

```diff
@@ -286,11 +286,11 @@ For example if `chain-config-dir` has the default value, then `config.json` can
 }
 ```
 
-For more information about C-Chain configs, see [here](command-line-interface.md#coreth-config).
+For more information about C-Chain configs, see [here](command-line-interface.md#config-file).
 
-### C-Chain / Coreth <a id="coreth-config"></a>
+### C-Chain / Coreth <a id="config-file"></a>
 
-`--coreth-config` \(json\):
+`--config-file` \(json\):
```

Comment:
This was correct the way it was before. `--coreth-config` and `--config-file` are different.

Reply:
Resolved